docker-compose.yml

```diff
@@ -26,3 +26,9 @@ services:
             - ./api/src:/api/src
         ports:
             - 3000:3000
+    he_twitter_miner:
+        build: .\he_twitter_miner
+        volumes:
+        - ${HOST_HE_TWITTER_MINER_SRC_DIR}\he_twitter_miner\src:/usr/src/app
+        - ${HOST_HE_TWITTER_MINER_SRC_DIR}\config:/usr/src/app/config
```

Comment:
have you tried this service on something other than windows? I feel like the paths are going to break on the server.